bin/next-init

```diff
@@ -12,14 +12,18 @@ const argv = parseArgs(process.argv.slice(2), {
 
 const dir = resolve(argv._[0] || '.')
 
-exists(join(dir, 'package.json'))
+exists(dir)
 .then(async present => {
+  if (!present) {
+    await mkdir(dir)
```

Comment:
Updated to remove the `^` from the `mkdirp-then` dependency to match rest of `package.json`.

I find it curious that you're pinning dependencies in your own `package.json`, yet recommend other people to use the most flexible semver possible with `"next": "*"` [(See Blog post)](1) or `"next": "latest"` [(See Readme)](2) in their `package.json`.

[1]: https://zeit.co/blog/next#simple-deployment
[2]: https://github.com/zeit/next.js#production-deployment